aocd/__init__.py

```diff
@@ -11,13 +12,34 @@
 from . import post
 from . import runner
 from . import utils
+from . import types
 from .exceptions import AocdError
 from .get import get_data
 from .get import get_day_and_year
 from .post import submit as _impartial_submit
 
+__all__ = [
+    "_ipykernel",
+    "cli",
+    "cookies",
+    "data",
+    "examples",
+    "exceptions",
+    "get",
+    "models",
+    "post",
+    "runner",
+    "submit",
+    "utils",
+    "types",
+]
 
-def __getattr__(name):
+if t.TYPE_CHECKING:  # pragma: no cover
```

Comment:
I don't like inline pragma comments - please move this to config i.e. tool subsection in the `pyproject.toml`

Reply:
I ... did not know you could do this! :-D